dashboard/src2/pages/NewBench.vue

```diff
@@ -208,10 +205,80 @@ export default {
 			};
 		}
 	},
+	methods: {
+		getAppsToInstall() {
+			let apps = [
+				this.options.versions
+					.find(v => v.name === this.benchVersion)
+					.apps.find(app => app.name === 'frappe')
+			].map(app => {
+				return {
+					name: app.name,
+					source: app.source.name
+				};
+			});
+
+			// add default apps
+			this.defaultApps.forEach(app => {
+				apps.push({
+					name: app.name,
+					source: app.source
+				});
+			});
+
+			return apps;
+		}
```

Comment:
We could avoid sending the apps along with source from frontend and instead compute the app and app source map from the backend. Would make the API and code simpler.